### ISSUE TITLE
The visualisation identifier should be distinct from the original one

### ISSUE THREAD

**Issue by justvanrossum** [BlackFoundryCom/fontra-reference-font#1]
Please replace this:
```
identifier: "fontra.reference.font",
```
with:
```
identifier: "fontra.reference-font-plugin",
```

This should then isolate the settings, and we can use the original panel and the plugin version at the same time, without them interfering with each other.

Please do this as a PR instead of directly committing to main.

**Comment by justvanrossum** [comment]
Actually, also
```
    this.controller.synchronizeWithLocalStorage("fontra.reference-font.");
```
should become
```
    this.controller.synchronizeWithLocalStorage("fontra.reference-font-plugin.");
```

**Comment by justvanrossum** [comment]
Fixed by #2.
